RecastDemo/Source/Sample_SoloMesh.cpp

```diff
@@ -435,7 +435,7 @@ bool Sample_SoloMesh::handleBuild()
 	// If your input data is multiple meshes, you can transform them here, calculate
 	// the are type for each of the meshes and rasterize them.
 	memset(m_triareas, 0, ntris*sizeof(unsigned char));
-	rcMarkWalkableTriangles(m_ctx, m_cfg.walkableSlopeAngle, verts, nverts, tris, ntris, m_triareas);
+	rcMarkWalkableTriangles(m_ctx, m_cfg.walkableSlopeAngle, verts, nverts, tris, ntris, m_triareas, rcAreaModification(SAMPLE_POLYAREA_GROUND));
```

Comment:
Lines 642 and 643 have been reworked, defining flags from area has been factored in function `sampleAreaToFlags`, reused in the 3 `Sample` implementations.